data/dictionary/items with key index getter/internal/a8bd43b5-92b7-4f0e-8915-6dab60d12439.json

```diff
@@ -0,0 +1,41 @@
+{
+  "description": "",
+  "interface": {
+    "inputs": {
+      "9af21210-8f9c-4067-986a-f29fe11cede0": {
+        "type": "{\"dict\":{string:any},\"keys\":string[]}",
+        "name": "dict & keys"
+      }
+    },
+    "outputs": {
+      "9e6164b8-807d-4f53-9a1b-2c8654bdcbbd": {
+        "type": "{string: any}",
+        "name": "items"
+      }
+    }
+  },
+  "implementations": {
+    "es6": {
+      "cleanupFunction": "",
+      "dependencies": {},
+      "initFunction": "",
+      "inputFunctions": {
+        "9af21210-8f9c-4067-986a-f29fe11cede0": "const {dict, keys} = data;\noutputs.values(keys.map(key => dict[key]), tag);"
+      }
+    },
+    "es6-node": {
```

Comment:
The `es6-node` implementation differs from the `es6` implementation. The former seems to be the older one.
I think we should only have an es6 implementation as there's nothing node or browser specific in the code.

Reply:
Done